src/Sulu/Bundle/WebsiteBundle/Routing/ContentRouteProvider.php

```diff
@@ -306,11 +304,16 @@ protected function getRedirectWebSpaceRoute(Request $request)
         $redirect = $this->urlReplacer->replaceLocalization($redirect, $localization->getLocale(Localization::DASH));
 
         // redirect by information from webspace config
+        // has to be done for all routes, because double slashes introduce problems otherwise
         return new Route(
-            $request->getPathInfo(), [
+            '/{wildcard}',
+            [
                 '_controller' => 'SuluWebsiteBundle:Redirect:redirectWebspace',
                 'url' => $this->requestAnalyzer->getPortalUrl(),
                 'redirect' => $redirect,
+            ],
+            [
+                'wildcard' => '.*',
             ]
```

Comment:
@wachterjohannes What do you think about that? The only solution I found to the problem explained in https://github.com/sulu/sulu/issues/2562#issuecomment-259144408